ses/client.bal

```diff
@@ -0,0 +1,519 @@
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# The Ballerina AWS SES connector provides the capability to access AWS Simple Email Service related operations.
+# This connector lets you to to send email messages to your customers.
+#
+# + awsSesClient - Connector HTTP endpoint
+# + accessKey - AWS SES access key ID
+# + secretKey - AWS SES secret access key
+# + securityToken - AWS SES Security token
+# + region - AWS SES Region
+# + host - AWS SES host name
+@display {label: "Amazon SES Client", iconPath: "resources/aws.ses.svg"}
+public isolated client class Client {
+
+    private final http:Client awsSesClient;
+    private final string accessKey;
+    private final string secretKey;
+    private final string? securityToken;
+    private final string region;
+    private final string host;
+    
+    # Initializes the connector. During initialization you have to pass access key id and secret access key
+    # Create an AWS account and obtain tokens following [this guide]
+    # (https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+    #
+    # + amazonSesconfiguration - The configurations to be used when initializing the client
+    # + httpConfig - HTTP Configuration
+    # + return - `http:Error` in case of failure to initialize or `null` if successfully initialized
+    public isolated function init(ClientConfiguration amazonSesconfiguration, http:ClientConfiguration httpConfig = {}) 
+                                  returns error? {
+        self.accessKey = amazonSesconfiguration.awsCredentials.accessKeyId;
+        self.secretKey = amazonSesconfiguration.awsCredentials.secretAccessKey;
+        self.securityToken = amazonSesconfiguration.awsCredentials?.securityToken;
+        self.region = amazonSesconfiguration.region;
+        self.host = AWS_SERVICE + DOT + self.region + DOT+ AWS_HOST;
+        string endpoint = HTTPS + self.host;
+        self.awsSesClient = check new (endpoint,httpConfig);
+    }
+
+    # Creates a contact list.
+    #
+    # + contactListCreationRequest - The request payload to create contact list
+    # + return - An empty json on success else an error
+    @display {label: "Create Contact List"}
+    remote isolated function createContactList(@display {label: "Contact List To Create"}
+                                               ContactListCreationRequest contactListCreationRequest)
+                                               returns @display {label: "Contact List Creation Response"} json|error {
```

Comment:
Can't we return as a record instead of json?

Reply:
It is removed since it's an empty response. Error is only returned.